app/controllers/api/category_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::CategoryController < ApplicationController
+  def index
+    @categories = Category.all
```

Comment:
Since the field isn't doing any work here, this could probably just be `render json: Category.all`.